tibiakt-core/src/test/kotlin/com/galarzaa/tibiakt/core/parsers/HousesSectionParserTests.kt

```diff
@@ -17,53 +17,61 @@
 package com.galarzaa.tibiakt.core.parsers
 
 import com.galarzaa.tibiakt.TestResources.getResource
-import com.galarzaa.tibiakt.core.enums.HouseOrder
 import com.galarzaa.tibiakt.core.enums.HouseStatus
-import com.galarzaa.tibiakt.core.enums.HouseType
-import io.kotest.core.spec.style.StringSpec
-import io.kotest.inspectors.forAll
+import com.galarzaa.tibiakt.core.models.house.HousesSection
+import io.kotest.core.spec.style.FunSpec
 import io.kotest.inspectors.forAtLeastOne
+import io.kotest.matchers.collections.shouldHaveAtLeastSize
 import io.kotest.matchers.collections.shouldHaveSize
+import io.kotest.matchers.nulls.shouldBeNull
 import io.kotest.matchers.shouldBe
 import io.kotest.matchers.shouldNotBe
+import io.kotest.matchers.types.shouldBeInstanceOf
 
-class HousesSectionParserTests : StringSpec({
-    "Parse house list"{
-        val housesSection = HousesSectionParser.fromContent(getResource("houses/houseList.txt"))
-        housesSection shouldNotBe null
-        housesSection!!.world shouldBe "Ardera"
-        housesSection.town shouldBe "Carlin"
-        housesSection.status shouldBe null
-        housesSection.type shouldBe HouseType.HOUSE
-        housesSection.order shouldBe HouseOrder.NAME
-        housesSection.entries shouldHaveSize 97
-        housesSection.entries.also {
-            it.forAll { house ->
-                house.type shouldBe housesSection.type
-                house.town shouldBe housesSection.town
-                house.world shouldBe housesSection.world
-            }
-            it.forAtLeastOne { house ->
-                house.status shouldBe HouseStatus.AUCTIONED
-                house.highestBid shouldBe 0
-                house.timeLeft shouldNotBe null
-            }
+class HousesSectionParserTests : FunSpec({
+    test("Houses section"){
+        val housesSection = HousesSectionParser.fromContent(getResource("housesSection/housesSection.txt"))
+
+        housesSection.shouldBeInstanceOf<HousesSection>()
+
+        with(housesSection){
+            world shouldBe "Alumbra"
+            town shouldBe "Carlin"
+            entries shouldHaveAtLeastSize 1
+        }
+    }
+    test("Houses section empty"){
```

Comment:
I recommend using ktlint. :smile:

Reply:
I will try it again, last time I tried I struggled a bit to set it up.